packages/mui-styled-engine/src/StyledEngineProvider/StyledEngineProvider.js

```diff
@@ -3,12 +3,41 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import { CacheProvider } from '@emotion/react';
 import createCache from '@emotion/cache';
+import { StyleSheet } from '@emotion/sheet';
+
+const createEmotionCache = (options) => {
```

Comment:
Let's add a link to the Emotion issue

```suggestion
// We might be able to remove this when this issue is fixed:
// https://github.com/emotion-js/emotion/issues/2790
const createEmotionCache = (options) => {
```